### PR TITLE
CMS/380 Create pug file for title page

### PR DESCRIPTION
**Pull request by foxyfiction**
Add title002.pug
Add title002.css
Add title002.js
Add title002Data.js
Add title002.jpg
Modify router.js

### CODE REVIEW

File: backend/app/controllers/title002.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const titleData = require('../../dataForRender/title002Data');
+
+module.exports = (request, response) => {
+    response.render('pages/title002', {
+        title: titleData.titleText[0],
+        describe: titleData.titleText[1]
+    });
+};
```
Comment (mvahrameev) [inline backend/app/controllers/title002.js]: Add new line

File: backend/app/views/pages/title002.pug [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        link(rel="stylesheet" href="css/title002.css")
+    body
+        .cover
+            .cover__background
+            .cover__filter
+            .container
+                .container-cell
+                        .title.cover__title
+                            h1= title
+                        .cover__describe.describe.describe_font-size= describe
```
Comment (mvahrameev) [inline backend/app/views/pages/title002.pug]: New line

File: backend/app/controllers/title002.js (hunk 1 above)
Comment (mvahrameev) [inline backend/app/controllers/title002.js]: Use sensible names for controllers, `title002` is not the best choice

File: backend/dataForRender/title002Data.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+let titleText = [
+    'Forest',
+    'The term "fog" is distinguished from the more generic term "cloud" in that fog is low-lying, ' +
+    'and the moisture in the fog is often generated locally.'
+];
+
+module.exports = {
+    titleText
+};
```
Comment (mvahrameev) [inline backend/dataForRender/title002Data.js]: New line

File: backend/dataForRender/title002Data.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+let titleText = [
+    'Forest',
```
Comment (maxmalov) [inline backend/dataForRender/title002Data.js]: You should consider using object to host title parts, so the object properties will indicate the kind of title part

File: backend/app/views/pages/title002.pug [hunk 5]
```diff
@@ -0,0 +1,16 @@
+doctype html
```
Comment (maxmalov) [inline backend/app/views/pages/title002.pug]: you should consider to extract common template out of the pages into a layout (a main block in terms of pug)

**Pull request closed without merging**